.github/workflows/ci.yml

```diff
@@ -109,10 +112,22 @@ jobs:
           - "7.4"
           - "8.0"
 
+        php-ini-values:
+          - assert.exception=1, zend.assertions=1
```

Comment:
With the previous configuration, we only override the entries for PHP 8.0 in the build matrix.

With this change, we now add two additional jobs for PHP 8.0 with different `php-ini-values`.